src/style.css

```diff
@@ -1 +1,22 @@
 /* styles go here */
+.title {
+  padding: 10px 0 ;
+}
+
+.button {
+  margin: 20px 0 ;
+  background-color: #fff ;
```

Comment:
Better practice is to use css variables for colors, that are used in multiple places.